backend/common/management/commands/clear_media_files.py

```diff
@@ -0,0 +1,27 @@
+import os
+import shutil
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+
+class Command(BaseCommand):
+    help = "Delete of data from the database"
+
+    def handle(self, *args, **options):
+        media_root = settings.MEDIA_ROOT
+
+        if not os.path.exists(media_root):
+            self.stdout.write("Media directory does not exist.")
+            return
+
+        self.stdout.write(f"Deleting all directories inside {media_root}...")
+
+        try:
+            for root, dirs, files in os.walk(media_root, topdown=False):
```

Comment:
You don't use files variable. Can u itererate on only root and dirs? Maybe instead of using 'files' use '_'. I can't say if in the future we will use this variable